libsolidity/parsing/Parser.cpp

```diff
@@ -667,6 +681,8 @@ ASTPointer<EnumDefinition> Parser::parseEnumDefinition()
 	ASTNodeFactory nodeFactory(*this);
 	expectToken(Token::Enum);
 	ASTPointer<ASTString> name = expectIdentifierToken();
```

Comment:
Why can't you use `currentLocation` here?

Reply:
done. thanks.